### DISCUSSION TITLE
Howto update a NextCloud Alpine instance

### DISCUSSION THREAD

**Discussion by Basti-Fantasti** [tteck/Proxmox#3722] [General] [answered]
The Alpine NextCloud Hub script does not contain an update functionality. 
In a script version on an older screenshot it seems there was an update option available.

What is the correct way to update the NextCloud version running inside this container?
The documentation says to run the install script, which is not sufficient, because no update option is available.

I also tried to set the version info in the config.php but without success.

I'm looking forward to some assistance

**Comment by tteck** [comment, marked as answer]
Alpine-Nextcloud can be updated via the Alpine Package Keeper (APK) if the package is available in the Alpine package repositories.

**Reply by Basti-Fantasti** [reply]
Thanks for your reply.

based on the package information a version 29.0.x should be available. Installed at the time of setting up the Container was 27.1.x
I will check and update the topic here accordingly.

**Reply by tteck** [reply]
Version 29 is available on Alpine 3.20
https://pkgs.alpinelinux.org/packages?name=nextcloud&branch=v3.20&repo=&arch=x86_64&maintainer=

**Reply by Basti-Fantasti** [reply]
Yes, the update installed now to v29 using the apk update / apk upgrade procedure, but now the Nextcloud does not come up due to an error with a module

```
Application files_reminders (files_reminders) is not present or has a non-compatible version with this server. Please check the apps directory.
```

I think I saw this issue discussed in another issue or discussion.

**Reply by Basti-Fantasti** [reply]
The update did crash the installation due to some issues with the database.
I'll roll back my installation to the latest backup and try again

**Reply by Basti-Fantasti** [reply]
Did anyone else successfully update the instance to v29?

**Reply by miloumeeloo** [reply]
I've updated successfully to v29 by using recommendation from step 3 of this post to solve database problem
https://help.nextcloud.com/t/unknown-column-lazy-in-field-list-when-upgrading-from-28-0-1-1-to-29-0-3-4/196263/10
